pkg/cstor/poolcluster/v1alpha1/build.go

```diff
@@ -43,6 +43,26 @@ func (b *Builder) WithName(name string) *Builder {
 	return b
 }
 
+// WithNamespace sets the Namespace field of CSPC with provided value.
+func (b *Builder) WithNamespace(namespace string) *Builder {
+	if len(namespace) == 0 {
```

Comment:
namespace we can check using `if namespace == ""{}`  instead of calculating the length

Reply:
Done